packages/tests/package.json

```
@@ -5,15 +5,19 @@
   "sideEffect": false,
   "type": "module",
   "scripts": {
-    "test": "playwright test",
-    "test:ui": "playwright test --ui",
+    "test": "NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test",
+    "test:ui": "NODE_OPTIONS=--disable-warning=ExperimentalWarning playwright test --ui",
     "lint": "eslint ."
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "^1.51.1",
-    "@types/node": "22.13.10"
+    "@playwright/test": "^1.51.0",
+    "@types/node": "22.7.7",
+    "knex": "^3.1.0",
+    "pg": "^8.13.3",
+    "playwright": "^1.51.0",
+    "testcontainers": "^10.18.0"
   }
```

Suggestion: We should build `cpu-features` dependencies for performance matters.


```suggestion
  },
  "onlyBuiltDependencies": [
      "cpu-features",
```